src/webgpu/api/validation/createRenderPipeline.spec.ts

```diff
@@ -187,3 +238,34 @@ g.test('sample_count_must_be_valid')
 
     t.doCreateRenderPipelineTest(isAsync, _success, descriptor);
   });
+
+g.test('pipeline_fragment_output_type_must_be_compatible_with_target_color_state_format')
```

Comment:
This can have a shorter name and have the details in the description. The really long test names are an artifact from before we had test descriptions.